src/cljs/main/broadfcui/common/table_utils.cljs

```diff
@@ -148,9 +148,12 @@
        [:div {:style (merge {:fontSize "80%" :fontWeight 500} (:body-style props))}
         (map-indexed
           (fn [row-index row]
-            (let [row-style (:row-style props)
-                  row-style (if (fn? row-style) (row-style row-index row) row-style)]
-              [:div {:style (merge {:display "flex" :alignItems "center"} row-style)}
+            (let [get-row-style #(if (fn? %) (% row-index row) %)
+                  row-style (-> props (:row-style) (get-row-style) (merge {:display "flex" :alignItems "center"}))
```

Comment:
Minor style thing: you can omit the parens around single entries in a threading macro.  So just `(-> props :row-style get-row-style ...)`.